data_steward/cdr_cleaner/cleaning_rules/truncate_fitbit_data.py

```diff
@@ -186,28 +179,26 @@ def validate_rule(self, client):
         raise NotImplementedError("Please fix me.")
 
     def get_sandbox_tablenames(self):
-        sandbox_date_tables = []
-        sandbox_datetime_tables = []
-        for table in FITBIT_DATE_TABLES:
-            sandbox_date_tables.append(
-                f'{self._issue_numbers[0].lower()}_{table}')
-        for table in FITBIT_DATETIME_TABLES:
-            sandbox_datetime_tables.append(
-                f'{self._issue_numbers[0].lower()}_{table}')
-        return sandbox_date_tables, sandbox_datetime_tables
+        sandbox_tables = []
+        for table in self.affected_tables:
+            sandbox_tables.append(self.sandbox_table_for(table))
+        return sandbox_tables
 
 
 if __name__ == '__main__':
     import cdr_cleaner.args_parser as parser
     import cdr_cleaner.clean_cdr_engine as clean_engine
 
     ext_parser = parser.get_argument_parser()
-    ext_parser.add_argument('-t',
-                            '--truncation_date',
-                            action='store',
-                            dest='truncation_date',
-                            help='CDR truncation date',
-                            required=True)
+    ext_parser.add_argument(
+        '-t',
+        '--truncation_date',
+        action='store',
+        dest='truncation_date',
+        help=
+        ('Cutoff date for data based on <table_name>_date and <table_name>_datetime fields.  '
```

Comment:
nit: This help needs an update - Column names are not `<table_name>_date`/`<table_name>_datetime` for FitBit.